### ISSUE TITLE
JUnit stack traces are invalid after clover inclusion

### ISSUE THREAD

**Issue by ethrbunny** [bmuschko/gradle-clover-plugin#61]
When I include clover (via bmuschko plugin -> gradle 2.4) and have failed JUnit tests I get stack traces with missing line / file info:

com.monitor.config.FileConfigTest > oldConfig FAILED
    org.yaml.snakeyaml.scanner.ScannerException at null:-1

In the test report I see lines like this:

```
at com.config.YamlConfiguration.load(YamlConfiguration.java:45)
at com.monitor.config.FileConfig.parseStringConfig(Unknown Source)
at com.monitor.config.FileConfigTest.__CLR3_2_0wcn9m841f(Unknown Source)
at com.monitor.config.FileConfigTest.oldConfig(Unknown Source)
```

**Comment by lakshgupta** [comment]
I am also facing the same problem for a build using Gradle 3.1 and running tests using the TestNg plugin.  Is there a workaround available? I thought of setting 'enabled' to false in the clover task based on an argument, but that also does not help in this case.

**Comment by Alex-Vol** [comment]
This was fixed with the pending PR#84. I will merge it soon when @bmuschko approves the changes.

**Comment by Alex-Vol** [comment]
Fixed with [PR#84](https://github.com/bmuschko/gradle-clover-plugin/pull/84)
